packages/sources/kaiko/src/config/index.ts

```diff
@@ -11,7 +11,7 @@ export const config = new AdapterConfig(
     API_ENDPOINT: {
       description: 'API endpoint for  KAIKO',
       type: 'string',
-      default: 'https://us.market-api.kaiko.io/v2/data/trades.v1',
+      default: 'https://us.market-api.kaiko.io/v2/data/trades.v2',
```

Comment:
If anyone has the `API_ENDPOINT` currently manually set to the V1 API rather than using the default, will that continue to work with these code changes?

Reply:
AFAIK, all of the params were present in the v1 endpoint, and worst case they are unused in our code or optional params in the request to kaiko.
so... 99% sure yes, but i'll sanity check to make sure.

Reply:
verified all looks good with the data products key (which works with v1 and v2 endpoints).
This is an unlikely edge case scenario but will leave a comment in the ticket.